nvflare/app_common/workflows/cyclic_ctl.py

```diff
@@ -33,6 +33,9 @@ def __init__(
         persistor_id="persistor",
         shareable_generator_id="shareable_generator",
         task_name="train",
+        task_check_period: float = 0.5,
+        persist_every_n_rounds: int = 1,
+        snapshot_every_n_rounds: int = 1,
     ):
```

Comment:
Need another arg for relay order:
Fixed - use the same order for all rounds
Random - randomize client order for each round
Random_No_Consecutive_Clients - randomize client order for each round AND not same client twice in a row (last of the last round and first of next round must be different)

Default is FIXED.

Reply:
thanks, fixed